app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
 
     defaultConfig {
         applicationId "pl.rspective.voucherify.android.demo"
-        minSdkVersion 10
```

Comment:
Do we really need increase it? Based on Google's stats this version is used by 2,2% what already gives some numbers and seems that this SDK doesn't require any features which requires to increase it.